modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -269,6 +269,43 @@ Card Shadow:: Specifies the shadow beneath the card.
 Card Width:: Specifies the card width.
 Card Height:: Specifies the card height.
 
+== Integrated Search Input
+
+The Integrated Search Input component enables users to search data integrated from both Community Cloud and Anypoint Platform.
+
+You can specify values for the following parameters of the Integrated Search Input:
+
+Component Title:: Specifies the title for the search input field.
+Number of Results - Instant Incremental Response:: Specifies the number of results shown as the user types a query.
+Search Areas:: Selects which types of content to search. Choices include APIs, Applications, Knowledge Articles, Cases, Pages, and Forums.
+Linked Result Page:: Specifies the URL of the search results page shown after the user enters a search.
+
+=== Search Configuration
+
+Search is a basic expected feature in any developer portal, and becomes important as the number of APIs grows to around 50 and beyond.
+
+The example portals and defaults have one Integrated Search Input and one Integrated Search Response Page. This is the best practice to minimize the chance of misconfigurations.
+
+The Integrated Search Input parameter Linked Result Page specifies the search results page. That page must have an Integrated Search Response Page component to display the search results.
+
+The Integrated Search Response Page component shows only results of the content types specified in the Search Areas parameters of the Integrated Search Input component. If a content type is not selected then the search results page does not display any search results of that content type.
+
+The current version of the integrated search does not include CMS articles in the search results.
```

Comment:
This needs to be called out with `IMPORTANT`.

Reply:
Done. Thank you!